.devcontainer/develop_on_containers.rst

```diff
@@ -0,0 +1,67 @@
+.. _develop_on_remote_containers:
+
+Develop on a remote container
+=============================
+
+.. note:: This approach requires to have a docker image with MAPDL installed.
+   The one listed in the 
+   `repository development container configuration files <pymapdl_devcontainer_configuration_>`_ 
+   is not for public usage due to licensing issues.
+   However, you are invited to `create your own docker image <ref_make_container_>`
+   and use this file as template for setting your own local development container.
+
+To use a `remote container <vscode_devcontainers_containers_>`_, you must install:
+
+* `VS Code <vscode_>`_
+* `Docker software <docker_main_>`_ or equivalent.
+  It is recommended to use Windows Subsystem Linux (WSL) backend to run Linux
+  docker containers.
+  See `Developing inside a Container Getting started <vscode_devcontainers_containers_getting_started_>`_
+  for more information.
```

Comment:
```suggestion
  To run Linxu Docker containers, you should use the Windows Subsystem Linux (WSL) backend. For more
  information, see `Developing inside a Container Getting started vscode_devcontainers_containers_getting_started_>`_.
```

Reply:
the following element is part of the current list. ;)